lib/dotnet-parser.js

```diff
@@ -0,0 +1,235 @@
+var fs = require('fs');
```

Comment:
can we use `const` and `let` instead?

Reply:
yeah, just noticed. changing.

Reply:
I thought you wrote - `we can't use const and let` - which is true because of node 4

Reply:
We need to refactor this. Maybe to `ts`. But I believe not as part of this PR.